ajax/libs/store.js/package.json

```diff
@@ -16,11 +16,9 @@
     "target": "git://github.com/marcuswestin/store.js.git",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "dist",
         "files": [
-          "store.js",
-          "store.min.js",
-          "store+json2.min.js"
+          "**/*"
```

Comment:
Ummm ...
I think the files can be used in web front-end is `*.min.js` is lib, because the lib use browserify & uglify-js to generate [`*.min.js`](https://github.com/marcuswestin/store.js/blob/v2.0.12/scripts/compile-builds.js#L39) files.
Also , the README said usage way in https://github.com/marcuswestin/store.js/tree/v2.0.12#installation, it's just mention the minified flies.
And we just need to add https://github.com/marcuswestin/store.js#list-of-default-builds these files, I think add `store.tests.min.js` in CDNJS is a little bit weird, also there is no tutorial mention that we need it.
What do you think?

Many thanks.

PS. Need to use excluded mode in `dist` as the basePath.

Reply:
@sufuf3 
Actually, I don't know...
I still can not get a clear view of the principles that one should follow in the cdnjs.
This rule was actually suggested by the PeterDaveHello as you can see above.
I agree with you thoughts but I just don't know what should I follow.

Thanks!

Reply:
OK.
Let's wait for @PeterDaveHello 's decision.

Reply:
@PeterDaveHello would you please help this pattern problem?
You said the pattern should be used `**/*` https://github.com/cdnjs/cdnjs/pull/12351#discussion_r157731405

But I think adding `store.tests.min.js` in CDNJS is a little bit weird, becuase there is no tutorial mention that we need it.
What do you think?

Many thanks.